src/reducers/groups-reducer.tsx

```diff
@@ -0,0 +1,52 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { getAllGroups, createGroup as createGroupService, updateGroup } from "../services/group-services";
+import groups from "../data/users/groupsData";
+
+const groupsSlice = createSlice({
+  name: 'groups',
+  initialState: groups,
```

Comment:
should we eventually have the initial state be empty and pull from the getGroups endpoint?